cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -750,43 +747,43 @@
     <property>
         <name>metrics.query.bind.address</name>
         <value>0.0.0.0</value>
-        <description>Host address where the metrics query server is started</description>
+        <description>Metrics query server host address</description>
     </property>
 
     <property>
         <name>metrics.query.bind.port</name>
         <value>45005</value>
-        <description>Port for metrics query server to listen on</description>
+        <description>Metrics query server bind port</description>
     </property>
 
     <property>
         <name>metrics.data.table.retention.resolution.1.seconds</name>
         <value>7200</value>
-        <description>Retention resolution 1 sec table in seconds</description>
+        <description>Retention resolution of the 1 second table in seconds; default retention period is 2 hours</description>
```

Comment:
I meant '1-second', '1-hour' is the resolution of the metrics tables. Just '1-second table' sounded confusing to me.